notebooks/official/experiments/get_started_with_model_experimentation_preprocessed_data.ipynb

```diff
@@ -0,0 +1,1176 @@
+{
```

Comment:
<p>explain what they should expect to see.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/558/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='60'/>